docs/process.rst

```diff
@@ -985,227 +1048,184 @@ Some caveats on glob pattern behavior:
-    path seq from sequences
+    val species
+    path seq
 
     output:
-    path "${x}.aln" into genomes
+    path "${species}.aln"
 
     """
-    t_coffee -in $seq > ${x}.aln
+    t_coffee -in $seq > ${species}.aln
     """
   }
 
-In the above example, each time the process is executed an alignment file is produced whose name depends
-on the actual value of the ``x`` input.
+In the above example, each process execution produces an alignment file whose name depends
+on the actual value of the ``species`` input.
 
 .. tip::
   The management of output files in Nextflow is often misunderstood.
+
   With other tools it is generally necessary to organize the output files into some kind of directory
   structure or to guarantee a unique file name scheme, so that result files don't overwrite each other
   and so they can be referenced unequivocally by downstream tasks.
 
   With Nextflow, in most cases, you don't need to manage the naming of output files, because each task is executed
-  in its own unique directory, so files produced by different tasks can not override each other.
+  in its own unique directory, so files produced by different tasks can't overwrite each other.
   Also, metadata can be associated with outputs by using the :ref:`tuple output <process-out-tuple>` qualifier, instead of
   including them in the output file name.
 
   To sum up, the use of output files with static names over dynamic ones is preferable whenever possible, 
   because it will result in simpler and more portable code.
 
 
-.. _process-out-path:
-
-Output path
------------
+.. _process-env:
 
-The ``path`` output qualifier was introduced by Nextflow version 19.10.0 and it's a drop-in replacement
-for the ``file`` output qualifier, therefore it's backward compatible with the syntax
-and the semantic for the input ``file`` described above.
+Output type ``env``
+-------------------
 
-The main advantage of ``path`` over the ``file`` qualifier is that it allows the specification
-of a number of outputs to fine-control the output files.
+The ``env`` qualifier allows you to output a variable defined in the process execution environment::
 
-============== =====================
-Name            Description
-============== =====================
-glob            When ``true`` the specified name is interpreted as a glob pattern (default: ``true``)
-hidden          When ``true`` hidden files are included in the matching output files (default: ``false``)
-followLinks     When ``true`` target files are return in place of any matching symlink (default: ``true``)
-type            Type of paths returned, either ``file``, ``dir`` or ``any`` (default: ``any``, or ``file`` if the specified file name pattern contains a `**` - double star - symbol)
-maxDepth        Maximum number of directory levels to visit (default: `no limit`)
-includeInputs   When ``true`` any input files matching an output file glob pattern are included.
-============== =====================
```

Comment:
This has been moved or dropped?

Reply:
it was moved further up

Reply:
sorry I probably should not have rearranged things, since then I am trying to minimize changes